travvy/src/components/AttractionsList.vue

```diff
@@ -85,35 +85,83 @@
       <h1>Map</h1> 
       <div id="map">
         <div v-if="$store.state.city == 'Toronto'">
-        <gmap-map v-bind:center="{lat: 43.6532, lng: -79.3832}" v-bind:zoom="11" style="height: 425px">
+        <gmap-map v-bind:center="{lat: 43.6532, lng: -79.3832}"
+          v-bind:zoom="11"
+          style="margin:45px; height: 425px;">
+          <GmapMarker
```

Comment:
We talked when you were having some trouble with value.lat and it was causing an error. So glad you were able to figure out how to fix this issue after we identified it! I think it is great to have these values that are dynamic and not hard coded :) Great job improving a feature we implemented in sprint 3, but improved to be how we had planned it to be in sprint 4